Tests/LibGL/TestRender.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2021, Leon Albrecht <leon2002.la@gmail.com>.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibTest/TestCase.h>
+
+#include <AK/Debug.h>
+#include <AK/Format.h>
+#include <LibGL/GL/gl.h>
+#include <LibGL/GLContext.h>
+#include <LibGfx/BMPWriter.h>
+#include <LibGfx/Bitmap.h>
+#include <LibGfx/FontDatabase.h>
+#include <fcntl.h>
+#include <unistd.h>
+
+#define RENDER_WIDTH 800
+#define RENDER_HEIGHT 800
```

Comment:
Just a general thought: We might make these dimensions smaller for the tests to run as quickly as possible. 10x10 or something should be plenty enough for simple test cases.

Reply:
should suffice for the current use-case, when we are actually testing the image, we should increase it again,
Some issues could be mere sub-pixels at that size, but we will roll with that for now then